include/azure-tools.bash

```diff
@@ -0,0 +1,22 @@
+azure-configure-arm() {
+    declare desc="Configure new ARM application"
+
+    docker run -it sequenceiq/azure-cli-tools:0.5 arm_configure \
+      --app_name ${1:? app_name is required as 1. param} \
+      --app_password ${2:? app_password is required as 2. param} \
+      --subscription_id ${3:? subscription_id is required as 3. param} \
+      --username ${4:? username is required as 4. param} \
+      --password ${5:? password is required as 5. param}
+}
+
+azure-deploy-dash() {
+    declare desc="Deploy the MicrosoftDX/DASH project to a cloud service and create its storage accounts "
+
+    docker run -it sequenceiq/azure-cli-tools:0.5 deploy_dash \
+      --accounts ${1:? accounts is required as 1. param} \
+      --prefix ${2:? prefix is required as 2. param} \
+      --location ${3:? location is required as 3. param} \
+      --instances ${4:? instances is required as 4. param} \
+      --username ${5:? username is required as 5. param} \
+      --password ${6:? password is required as 6. param}
```

Comment:
I think it would be good if we could have default values (the deployer tool has them), so these won't be required parameters (maybe location should be required).
Username and password are not required to be provided with a switch  - these can be typed when prompted by the CLI so they won't be there in bash history.